tests/test_serializers.py

```diff
@@ -27,68 +27,134 @@ def setUp(self):
         self.invalid_hexa_short_color1 = "#12345"
         self.invalid_hexa_short_color2 = "#123R"
 
+        self.valid_rgb_colors = [
+            "rgb(5, 3, 6)",
+            "rgb(5,3,6)",
+            "rgb(50, 30, 128)",
+            "rgb(45,10, 99)",
+            "rgb(128, 128, 128)",
+            "rgb(156,200,250)",
+        ]
+        self.invalid_rgb_colors = [
+            "rgb(5)",
+            "rgb(5, 6)",
+            "rgb(128,255)",
+            "rgb(128,A)",
+            "rgb(128, 128, A)",
+            "rgb(128,255,12,30)",
+            "rgb(128, 255, 12, A)",
+        ]
+
+        self.valid_rgba_colors = [
+            "rgba(5, 3, 6, 1)",
+            "rgba(5,3,6,1)",
+            "rgba(50, 30, 128, 0.5)",
+            "rgba(45,10, 99, 0.6)",
+            "rgba(128, 128, 128, 0.3)",
+            "rgba(156,200,250,0)",
+        ]
+        self.invalid_rgba_colors = [
+            "rgba(10)",
+            "rgba(50, 100)",
+            "rgba(120,199)",
+            "rgba(1, 2, 3)",
+            "rgba(128,255,127)",
+            "rgba(128,255,127,A)",
+            "rgba(128, 255, 127, A)",
+            "rgba(128, 12, 35, 1, 1)",
+            "rgba(128, 12, 35, 1, 1, 2)",
+        ]
+
     def test_valid_serializer(self):
         data = {"color": self.valid_hex_full_color}
-        serializer = SerializerWithHexColor(data=data)
+        serializer = SerializerWithColor(data=data)
         self.assertTrue(serializer.is_valid())
 
         data = {"color": self.valid_hex_short_color}
-        serializer = SerializerWithHexColor(data=data)
+        serializer = SerializerWithColor(data=data)
         self.assertTrue(serializer.is_valid())
 
         data = {"color": self.valid_hexa_full_color}
-        serializer = SerializerWithHexColor(data=data)
+        serializer = SerializerWithColor(data=data)
         self.assertTrue(serializer.is_valid())
 
         data = {"color": self.valid_hexa_short_color}
-        serializer = SerializerWithHexColor(data=data)
+        serializer = SerializerWithColor(data=data)
         self.assertTrue(serializer.is_valid())
 
+        for value in self.valid_rgb_colors:
+            with self.subTest(f"RGB={value}"):
+                data = {"color": value}
+                serializer = SerializerWithColor(data=data)
+                self.assertTrue(serializer.is_valid())
+
+        for value in self.valid_rgba_colors:
+            with self.subTest(f"RGBA={value}"):
+                data = {"color": value}
+                serializer = SerializerWithColor(data=data)
+                self.assertTrue(serializer.is_valid())
```

Comment:
I've changed a bit the approach compared to the existing tests. I didn't refactor the existing ones as it felt out of scope of this PR.

Reply:
That's ok, the changes you did are good to me!